src/grimp/application/ports/graph.py

```diff
@@ -9,6 +9,11 @@
 from grimp.domain.valueobjects import Layer
 
 
+class Import(TypedDict):
```

Comment:
I'm wondering about the name here - the name is to be contrasted with the `DetailedImport` typed dict a few lines below. If "with line number and contents" is a `DetailedImport` then "without line number and contents" is just an `Import`? Other ideas might be `CompactImport` or `ImportSummary`.

Reply:
> Should DetailedImport inherit Import?

Done